### PR TITLE
Adding editorconfig and re-aligning code style with Modernizr

### LINKED ISSUE #8: Code style
It'd make sense if we followed Modernizrs coding guidelines basically convert the tabs to 2 spaces. I could do a PR that is just massive white space change along with adding an `.editorconfig` file

### PR DESCRIPTION
**Pull request by ryanseddon**
Fixes #8. 

Converts tabs to 2 spaces and adds editorconfig make sure you add the plugin to your favourite editor so it picks up the defaults.

### CODE REVIEW

File: src/js/stores/Results.js [hunk 1]
```diff
@@ -2,209 +2,209 @@ var AppDispatcher = require('../dispatcher/AppDispatcher');
 var EventEmitter = require('events').EventEmitter;
 var merge = require('react/lib/merge');
 var MetadataStore = require('./Metadata');
-var Fuse = require('../../../bower_components/fuse/src/fuse');
+var Fuse = require('fuse');
```
Comment (ryanseddon) [inline src/js/stores/Results.js]: This wasn't supposed to make it in but it fixes the build from being broken. Webpack couldn't resolve that path and you've added `bower_components` to the `moduleDirectories` so webpacks require resolving algorithm will check there already.

**Comment by patrickkettner** [comment]
looks good, assuming the fuse stuff dosen't break it.

**Comment by ryanseddon** [comment]
I couldn't build with the previous fuse path, had to change it to what's in this PR to work.

**Comment by patrickkettner** [comment]
hmm, thats a direct revert of @joecritch's c6476f718e57191e7b5c287d5f6c39cba4787cfc

**Comment by ryanseddon** [comment]
ok I'll remove that and we can discuss in another pr

**Comment by ryanseddon** [comment]
Reverted now this is straight formatting only

**Comment by patrickkettner** [comment]
[confirmed](https://github.com/Modernizr/modernizr-ui/pull/9/files?w=1)

**Pull request merged**